### ISSUE TITLE
Question: A few progress bars

### ISSUE THREAD

**Issue by IsFilimonov** [michaeladams/obsidian-text-progress-bar#12]
Hi. Is it possible to specify multiple sequences within a single code block `text-progress-bar`? I know it's not now, but is it planned in the near future?

For example:
```
\```text-progress-bar
Player 1:10/10
Player 2:3/2
fill:⚽
\```
```

At now:
```
\```text-progress-bar
Player 1:10/10
fill:⚽
\```
```
and
```
\```text-progress-bar
Player 2:3/2
fill:⚽
\```
```

**Comment by michaeladams** [comment]
Hi - that's a pretty good idea!  I was actually wondering how to represent things like months over a year, but having a series like you mention would probably do nicely for that use case too.

I've noted it as a todo, and see if I can tackle soonish :)

**Comment by michaeladams** [comment]
This has now been released as part of 0.2.0 and can be used by updating your plugin in Obsidian, with an example at the end of the readme.
